src/app/modules/verification/verification.controller.ts

```diff
@@ -332,7 +372,7 @@ export const _handleGenerateOtp: ControllerHandler<
 /**
  * Handler for the POST /forms/:formId/fieldverifications/:transactionId/fields/:fieldId/otp/generate endpoint
  */
-export const handleGenerateOtp = [
+export const handleGenerateFormOtp = [
```

Comment:
Is there a reason for renaming this and `handleOtpVerification`? 

If the reason was to distinguish between login OTPs and Form OTPs, then we also need to avoid the use of the word "form" in `FormFields` (in distinguishing from `PaymentField`). Since in one context, we are using `Form` to refer to both the `form_fields` and `paymentField`, and in the other we are using `Form` to distinguish between the two.

Reply:
Oh! This was from when I had both `handleGenerateFormOtp` and `handleGeneratePaymentOtp` for the 2 endpoints I set up initially. Will change it back!

Reply:
Okie, then can ignore all my other comments re the "form" prefix haha

Reply:
done in cb89e407